### ISSUE TITLE
regression_1039 fails with CFG_ENCRYPT_TA=y

### ISSUE THREAD

**Issue by jforissier** [OP-TEE/optee_test#661]
QEMUv8
```
$ make -j$(nproc) CFG_ENCRYPT_TA=y run
[...]
buildroot login: root
# xtest regression_1039
[...]
* regression_1039 Test subkey verification
o regression_1039.1 Load TA with two levels of subkeys
regression_1000.c:3206: xtest_teec_open_session(&session, &subkey1_ta_uuid, ((void *)0), &ret_orig) has an unexpected value: 0xffff000f = TEEC_ERROR_SECURITY, expected 0x0 = TEEC_SUCCESS
  regression_1039.1 FAILED
```

**Comment by jforissier** [comment]
The load is failing here https://github.com/OP-TEE/optee_os/blob/9901df47d96ff3117ee4abc4d3fd5afb001088b5/core/kernel/ree_fs_ta.c#L439

**Comment by jenswi-linaro** [comment]
Should be fixed with https://github.com/OP-TEE/optee_os/pull/5938

**Comment by jforissier** [comment]
> Should be fixed with [OP-TEE/optee_os#5938](https://github.com/OP-TEE/optee_os/pull/5938)

Indeed, thanks!
